# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/relations/browser.toml
# Commit: 4544c8e0eda1
# Full Hash: 4544c8e0eda1a54158ce635db02c449fa4e4338e
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-30 17:19 +0000
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P1: Implement details relationship for CSS anchored local accessibles. r=Jamie,emilio
# ==============================================================================

diff -r b8522d57b4e6 -r 4544c8e0eda1 accessible/tests/browser/relations/browser.toml
--- a/accessible/tests/browser/relations/browser.toml	Tue Sep 30 16:58:04 2025 +0000
+++ b/accessible/tests/browser/relations/browser.toml	Tue Sep 30 17:19:31 2025 +0000
@@ -15,6 +15,8 @@
   "layout.css.anchor-positioning.enabled=true"
 ]
 
+["browser_anchor_positioning.js"]
+
 ["browser_popover_and_command.js"]
 
 ["browser_relations_general.js"]